client/src/services/studentRequest.api.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+const SERVER_URL = "http://localhost:3001";
```

Comment:
Maybe it's better to read the server URL from a global place so that we don't repeat it in several files.